update from week 45

LC43 opened the pull request:
## not used:

* http://vrlist.io/experience/a-blast - gallery using aframe
* https://twitter.com/Stargayte/status/823652556605915136- video of a stargate made with a-frame
* http://virtuleap.vrsites.com/main.php - video player
* https://github.com/AdaRoseEdwards/aframe-verlet-worker-component - not yet released?
* http://www.westpierstudio.com/editorial/pokemon-2/
* https://vatelier.net/MyDemo/MozillaFOSDEM/ - map to  fosdem with a-frame
* [A-Frame 101](http://delapuente.github.io/presentations/aframe-101)

LC43 commented:
Hi @ngokevin ! thanks for the review. 

had a hard time deciding which ones made the cut, but I'll be tougher! :) 

One sec, and i'll commit those recommendations

LC43 commented:
pr updated :dancer:

ngokevin commented on README.md:
```diff
@@ -76,6 +76,7 @@ Learn about A-Frame.
 - [Porting a Water Shader to A-Frame for the Track Demo](https://medium.com/samsung-internet-dev/i-made-my-first-pr-to-a-frame-3675d596a2d8)
 - [Interactive 360 Video: Accurate Positioning and Smooth Tracking](https://corp.hapyak.com/resources/interactive-360-videos-accuracy/) - Interactive 360 videos by overlaying DOM elements and live editing 360 videos
 - [Hands-on With Virtual Reality Using A-Frame, React and Redux](https://medium.com/immersion-for-the-win/hands-on-with-virtual-reality-using-a-frame-react-and-redux-bc66240834f7)
+- [The Bow & Arrow is Virtual Reality’s “Hello, World”](https://medium.com/@jamesbpollack/the-bow-arrow-is-virtual-realitys-hello-world-b0556faa3ef8) - Building a Bow & Arrow game for tracked controllers
```
Could just be `Building a bow and arrow for tracked controllers`

ngokevin commented on README.md:
```diff
@@ -180,6 +181,9 @@ curated components, see the [Registry](https://aframe.io/aframe-registry).
 - [UI Components](https://github.com/gmarty/aframe-ui-components) - A collection of components related to UI
 - [Pipe](https://github.com/jellix79/aframe-pipe-component) - A pipe with inner and outer radius
+- [Persist Component](https://github.com/Utopiah/aframe-persist-component) - Use localStorage to make data persist over experiences
+- [Gradient Sky](https://github.com/zcanter/aframe-gradient-sky) - Use a gradient on sky
```
Could be a bit more descriptive (two colors, creates a gradient shader, colorful skies)

ngokevin commented on README.md:
```diff
@@ -180,6 +181,9 @@ curated components, see the [Registry](https://aframe.io/aframe-registry).
 - [Pipe](https://github.com/jellix79/aframe-pipe-component) - A pipe with inner and outer radius
+- [Persist Component](https://github.com/Utopiah/aframe-persist-component) - Use localStorage to make data persist over experiences
+- [Gradient Sky](https://github.com/zcanter/aframe-gradient-sky) - Use a gradient on sky
+- [Mouse Cursor Component](https://github.com/mayognaise/aframe-mouse-cursor-component) - Add interaction to the mouse cursor
```
Could be a bit more descriptive (uses hardware mouse, raycasts from XY position on canvas)

ngokevin commented:
some more nits, thanks!

LC43 commented:
Hi @ngokevin , i made the changes you suggested, fixed broken links and removed trailing slash from urls pointed by the linter. 2 of those didn't redirect to the correct links. on the last commit, i've used their shorturls, is that ok?

ngokevin commented:
Thanks!
